backend/src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,7 @@
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=user
+spring.datasource.password=password
```

Comment:
This may be OK for dev, but we should avoid committing passwords in general.
How is the production database authentication handled?

Reply:
The production passwords are stored in the config evironment in heroku. They are build into the msql url, so mysql:<username>:<password>@<dbaddress>.